panorama_connector.py

```diff
@@ -304,11 +304,13 @@ def _make_rest_call(self, data, action_result):
         config = self.get_config()
 
         try:
-            response = requests.post(self._base_url, data=data, verify=config[phantom.APP_JSON_VERIFY], timeout=DEFAULT_TIMEOUT)
+            response = requests.post(
+                self._base_url, data=data, verify=config[phantom.APP_JSON_VERIFY], timeout=DEFAULT_TIMEOUT)
         except Exception as e:
             self.debug_print(PAN_ERR_DEVICE_CONNECTIVITY, e)
-            return action_result.set_status(phantom.APP_ERROR, PAN_ERR_DEVICE_CONNECTIVITY,
-                                            self._get_error_message_from_exception(e))
+            return (action_result.set_status(phantom.APP_ERROR, PAN_ERR_DEVICE_CONNECTIVITY,
```

Comment:
you don't need parentheses to return multiple values btw

Reply:
Since this line spans multiple lines, I just thought using `parentheses` will make it easier to read (i.e. avoid see the `\` one)